sdk/python/kfp/_client.py

```diff
@@ -28,11 +28,8 @@
 class Client(object):
```

Comment:
The changes to this class probably belong to a different PR.

Reply:
excluded